tests/h/views/client_test.py

```diff
@@ -12,23 +12,6 @@
 
 
 @annotator_token_fixtures
-def test_annotator_token_calls_check_csrf_token(pyramid_request, session):
-    client.annotator_token(pyramid_request)
-
-    session.check_csrf_token.assert_called_once_with(pyramid_request)
-
-
-@annotator_token_fixtures
-def test_annotator_token_raises_Unauthorized_if_check_csrf_token_raises(
-        pyramid_request,
-        session):
-    session.check_csrf_token.side_effect = exceptions.BadCSRFToken
-
-    with pytest.raises(httpexceptions.HTTPUnauthorized):
```

Comment:
Ditto with the unused imports. You also have an unused `session` fixture in this file now.